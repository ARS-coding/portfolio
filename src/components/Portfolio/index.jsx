import React from 'react';
import { Col, Row } from 'react-bootstrap';
import { VerticalTimelineElement } from 'react-vertical-timeline-component';

const Portfolio = () => (
  <VerticalTimelineElement
    date="Nov 2020"
    dateClassName="date"
    className="vertical-timeline-element--work"
    contentStyle={{
      background: 'white',
      color: 'white',
    }}
    contentArrowStyle={{
      borderRight: '7px solid  white',
    }}
    iconStyle={{
      background: 'white',
    }}
    icon={<i className="fas fa-project-diagram projectIcon"></i>}
  >
    <div className="homeText projectContent">
      <h4>Portfolio</h4>
      <small>Solo Project</small> <br />
      <a
        href="https://github.com/tmansuroglu/portfolio"
        target="_blank"
        rel="noreferrer"
      >
        Github
      </a>
      <br />
      <p className="projectText">
        <strong>Goal:</strong> building a portfolio website to create an online
        presence.
      </p>
      <br />
      <Row>
        <Col xs={12} md={6}>
          <h5>Skills Applied</h5>
          <ul>
            <li>HTML</li>
            <li>CSS</li>
            <li>JavaScript</li>
            <li>React</li>
            <li>Emailjs</li>
            <li>Bootstrap</li>
            <li>Responsive Design</li>
            <li>Git and Github</li>
            <li>Testing (Jest)</li>
          </ul>
        </Col>
      </Row>
    </div>
  </VerticalTimelineElement>
);

export default Portfolio;
