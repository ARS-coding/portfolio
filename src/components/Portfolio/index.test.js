import React from 'react';
import renderer from 'react-test-renderer';
import Portfolio from '.';

it('Portfolio renders correctly', () => {
  const tree = renderer.create(<Portfolio />).toJSON();
  expect(tree).toMatchSnapshot();
});
