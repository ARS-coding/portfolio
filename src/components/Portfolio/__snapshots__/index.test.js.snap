// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Portfolio renders correctly 1`] = `
<div
  className="vertical-timeline-element--work vertical-timeline-element"
  id=""
  style={null}
>
  <span
    className="vertical-timeline-element-icon is-hidden"
    onClick={null}
    style={
      Object {
        "background": "white",
      }
    }
  >
    <i
      className="fas fa-project-diagram projectIcon"
    />
  </span>
  <div
    className="vertical-timeline-element-content is-hidden"
    onClick={null}
    style={
      Object {
        "background": "white",
        "color": "white",
      }
    }
  >
    <div
      className="vertical-timeline-element-content-arrow"
      style={
        Object {
          "borderRight": "7px solid  white",
        }
      }
    />
    <div
      className="homeText projectContent"
    >
      <h4>
        Portfolio
      </h4>
      <a
        href="https://github.com/tmansuroglu/portfolio"
        rel="noreferrer"
        target="_blank"
      >
        Github
      </a>
      <br />
      <p>
        <strong>
          Goal:
        </strong>
         building a portfolio website to create an online presence.
      </p>
      <br />
      <div
        className="row"
      >
        <div
          className="col-md-6 col-12"
        >
          <h5>
            Skills Applied
          </h5>
          <ul>
            <li>
              HTML
            </li>
            <li>
              CSS
            </li>
            <li>
              JavaScript
            </li>
            <li>
              React
            </li>
            <li>
              Emailjs
            </li>
            <li>
              Bootstrap
            </li>
            <li>
              Responsive Design
            </li>
            <li>
              Git and Github
            </li>
            <li>
              Testing (Jest)
            </li>
          </ul>
        </div>
        <div
          className="col-md-6 col-12"
        >
          <h5>
            My Responsibilities
          </h5>
          <ul>
            <li>
              Solo Project
            </li>
          </ul>
        </div>
      </div>
    </div>
    <span
      className="date vertical-timeline-element-date"
    >
      Nov 2020
    </span>
  </div>
</div>
`;
